stable/fluent-bit/templates/config.yaml

```diff
@@ -106,7 +106,11 @@ data:
 {{- end }}
 {{- if .Values.backend.es.http_user }}
         HTTP_User {{ .Values.backend.es.http_user }}
+{{- if .Values.backend.es.http_passwd }}
```

Comment:
password should be in a secret not in a configmap

Reply:
And it is in a secret. `.Values.backend.es.http_passwd` is the old option to provide the password in values, which I kept for backward compatibility.